tls/gen-cert.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
```

Comment:
It will be nice to add some comment about the purpose of this script (and note in the README file how to use it)

Reply:
Yep, I like the idea. I added how to use it in the README file.